### PR TITLE
AC New Automated Test - LRAC-11119 | master

### PR DESCRIPTION
**Pull request by joshitagaki**
Story: https://issues.liferay.com/browse/LRAC-10405
Ticket: https://issues.liferay.com/browse/LRAC-11119
Ignore due to bug: https://issues.liferay.com/browse/LRAC-11153

### CODE REVIEW

**Comment by joshitagaki** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase [hunk 1]
```diff
@@ -185,6 +185,226 @@ definition {
...
+
+			Close.closeWindow();
+		}
+
+		task ("Go to Forms Tab, check empty state message") {
+			ACNavigation.switchTab(tabName = "Forms");
+
+			ACUtils.viewGenericText(textValueList = "There are no visitors data found.,Check back later to verify if data has been received from your data sources, or you can try a different date range.,Learn more about forms.");
+
+			ACUtils.clickGenericHyperlink(hyperlinkText = "Learn more about documents and media.");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase:251]: Hey @joshitagaki, LGTM for automating this test. Only one minor adjustment, I think the hyperlink is "Learn more about forms." yes? After this fix, your code is very good for me.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase [hunk 2]
```diff
@@ -185,6 +185,226 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: LRAC-10698 | Automation ID: LRAC-11119 | Test Summary: Empty state of no assets with data source and property"
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase:188]: Hey @joshitagaki, I think this test is from this story [LRAC-10405](https://issues.liferay.com/browse/LRAC-10405)

`Feature ID: LRAC-10405`

**Review by christianstokes — APPROVED** [review]
Looks good to me, thank you!

**Comment by ViniLopes20** [comment]
ci:forward

**Pull request closed without merging**